### PR TITLE
Add missing swagger comments

### PR DESCRIPTION
**Pull request by petterroea**

### CODE REVIEW

File: backend/Controllers/ExperiencePostController.cs [hunk 1]
```diff
@@ -18,6 +18,9 @@
 
 namespace OpenData.Controllers
 {
+	/// <summary>
+	/// Returns a list of experience posts
```
Comment (Letaguator) [inline backend/Controllers/ExperiencePostController.cs]: Hmm, is this correct though?

**Comment by Letaguator** [comment]
Looks good to me, look at the comment above though.

I also see that you do not use the <returns></returns> and <param></param> as seen here https://github.com/OpenDataNTNU/OpenData/blob/master/backend/Controllers/UserController.cs. Is this an active decision, if so I am fine with it.

**Comment by petterroea** [comment]
What do you mean by "and as seen here"?

**Comment by Letaguator** [comment]
Look at line 76 and 77 on the link

**Comment by AndreasJJ** [comment]
How's the status on this pull request? are we waiting with it until the end of the sprint?

**Comment by petterroea** [comment]
@Letaguator please check now, it should be gucci

File: backend/Controllers/MetadataController.cs [hunk 2]
```diff
@@ -61,8 +65,10 @@ public async Task<IEnumerable<MetadataResource>> GetAllAsync()
 		}
 
 		/// <summary>
-		/// Returns a single metadata type, and all its associated metadata entries.
+		/// Returns a single metadata object, and some more granular information than what is retured by `/api/Matadata/`
```
Comment (Letaguator) [inline backend/Controllers/MetadataController.cs]: Small type: retured

**Pull request merged**